src/pid_design.jl

```diff
@@ -302,3 +302,50 @@ function loopshapingPI(P,ωp; ϕl=0,rl=0, phasemargin = 0, doplot = false)
     end
     return kp,ki,C
 end
+
+"""
+    piparams, C = placePI(P, ω₀, ζ; form=:standard)
+
+Selects the parameters of a PI-controller such that the poles of 
+closed loop between `P` and `C` are placed to match the poles of 
+`s^2 + 2ζω₀ + ω₀^2`.
+
+The form keyword allows you to choose which form the PI parameters
+should be returned on. 
+* `:standard` - `Kp*(1 + 1/Ti/s)`
+* `:series` - `Kc*(1 + 1/τi/s)`
+* `:parallel` - `Kp + Ki/s`
+* `:Ti` - `Kp + 1/(s*Ti)`
```

Comment:
```suggestion
* `:Ti` - `Kp + 1/(s*Ti)`   (non-standard form sometimes used in industry)
```

A bit confusing that it is named as a parameter. Or if we call it :nonstandard (which is a quite polite thing to call it)

Reply:
Yeah, was thinking about that. Took it from bagges exampel above. Maybe switch to :parallel_time or something similar?